content/en/blog/_posts/2022-11-11-retroactive-default-storage-class.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: blog
+title: "Retroactive default StorageClass"
+date: 2022-11-11
+slug: retroactive-default-storage-class
+---
+
+**Author:** Roman Bednář (Red Hat)
+
+In Kubernetes 1.25 a new Alpha feature was introduced to improve the way default StorageClass is assigned to a PersistentVolumeClaim (PVC). It is no longer required to create a default StorageClass first and PVC second in order to assign the class and any PVCs without StorageClass can be updated later. The feature will be Beta in Kubernetes 1.26.
+
+See [Official Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retroactive-default-storageclass-assignment) for more details.
```

Comment:
```suggestion
You can read [retroactive default StorageClass assignment](/docs/concepts/storage/persistent-volumes/#retroactive-default-storageclass-assignment) in the Kubernetes documentation for more details about how to use that,
or you can read on to learn about why the Kubernetes project is making this change.
```

Reply:
The above suggestion needs to be incorporated, please.

Reply:
> Also, would highly recommend that we use either nil or null even though they are the same and have the potential to confuse users.

Yes, it got a bit mixed up when applying suggestions. I believe @sftim mentioned we stick to `null` here: https://github.com/kubernetes/website/pull/37625#discussion_r1028495144

I've removed any references to `nil`.